src/containers/NdlaFilm/NdlaFilmEditor.jsx

```diff
@@ -107,7 +107,9 @@ class NdlaFilmEditor extends React.Component {
 
   fetchAllMovies = async () => {
     const contextType =
-      config.ndlaEnvironment === 'ff' ? 'standard' : 'topic-article';
+      config.ndlaEnvironment === 'ff' || config.ndlaEnvironment === 'staging'
```

Comment:
Fordi vi har importert data fra ff til staging må vi sjekke på begge disse for at ndlafilm skal funke.